src/ui/TimerView.tsx

```diff
@@ -252,13 +253,16 @@ export class TimerView extends React.Component<Props, State> {
             if (typeof e.data === "string") {
                 const [command, ...args] = e.data.split(" ");
                 switch (command) {
-                    case "start": this.start(); break;
+                    case "starttimer": this.start(); break;
+                    case "getcurrenttime": this.getTime(); break;
+                    case "getcurrenttimerphase": this.getTimerPhase(); break;
                     case "split": this.split(); break;
                     case "splitorstart": this.splitOrStart(); break;
                     case "reset": this.reset(); break;
-                    case "togglepause": this.togglePauseOrStart(); break;
-                    case "undo": this.undoSplit(); break;
-                    case "skip": this.skipSplit(); break;
+                    case "pause": this.togglePauseOrStart(); break;
```

Comment:
I'd prefer this to either stay `togglepause` or for `pause` to be a (deprecated) alias.

Reply:
Yeah that's a good idea, I'll fix that up :+1: